repos/system_upgrade/el7toel8/libraries/repofileutils.py

```diff
@@ -50,8 +64,8 @@ def get_parsed_repofiles(context=mounting.NotIsolatedActions(base_dir='/')):
     :rtype: List(RepositoryFile)
     """
     repofiles = []
-    cmd = ['find', '/etc/yum.repos.d/', '-type', 'f', '-name', '*.repo']
-    repofiles_paths = context.call(cmd, split=True)['stdout']
+    cmd = ['find'] + get_repodirs() + ['-maxdepth', '1', '-type', 'f', '-name', '*.repo']
+    repofiles_paths = context.call(cmd, split=True, checked=False)['stdout']
```

Comment:
why  `checked=False`? The change is not documented anywhere and I do not see reason for this to be honest.

Reply:
The difference is that if some of the directories is missing - which, by default, 2 out of 3 are - find returns 1 as exit code. This crashes the workflow.

Reply:
From that point, could you rather modify the `get_repodirs` function to return only existing paths instead?

Reply:
Yeah, I could do that.